### ISSUE TITLE
Fails on require - OSX

### ISSUE THREAD

**Issue by tehnorm** [ezequielgarcia/node-ao#2]
When trying to run the example:

```
var libao = require('libao');
// Create the libao instance
var ao = new libao({
  channels: 2,          // 2 channels
  bitDepth: 16,         // 16-bit samples
  sampleRate: 44100     // 44,100 Hz sample rate
});

// PCM data from stdin gets piped into the speaker
process.stdin.pipe(ao);
```

```
$ cat sounds/397435__foolboymedia__crowd-cheer-ii.wav | node index.js
dyld: lazy symbol binding failed: Symbol not found: _ao_initialize
  Referenced from: /Users/norm/Projects/bw/break-time/node_modules/libao/build/Release/binding.node
  Expected in: flat namespace

dyld: Symbol not found: _ao_initialize
  Referenced from: /Users/norm/Projects/bw/break-time/node_modules/libao/build/Release/binding.node
  Expected in: flat namespace

Abort trap: 6
```

**Comment by ezequielgarcia** [comment]
Thanks for the report! I don't have OSX to test easily.

1. Can you print the symbols in your binding.node and libao library (should be libao.dylib?). I think you could use otool -TV.
2. Can you print the libraries linked to binding node? You could use otool -L.

**Comment by tehnorm** [comment]
Not a problem. 

symbols in your binding.node are [binding-node-dump.txt](https://github.com/ezequielgarcia/node-libao/files/1156566/binding-node-dump.txt). Where would the `libao.dylib`

 be located? It didn't seem to be in `~/Projects/bw/break-time/node_modules/libao`

libraries linked to binding node
```
norm ~/Projects/bw/break-time/node_modules/libao/build/Release (master) $ otool -L binding.node
binding.node:
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.5.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
```

**Comment by ezequielgarcia** [comment]
@tehnorm Sorry for the delay! I have updated the build stuff and published 0.0.4. It is supposed to work on OSX (tested on El Capitan). Please give it a shot and re-open the ticket if still failing.
